src/pages/Home/index.js

```diff
@@ -1,9 +1,10 @@
 import React, { useEffect } from 'react';
-import ajax from '../../utils/ajax';
+import ajax from 'utils/ajax';
+import { ACCESS_TOKEN } from '../../constants';
 
 const Home = () => {
   useEffect(() => {
-    ajax.get("http://mock.don.red/weibo/2/statuses/public_timeline.json", { params: { access_token: 'token' }})
```

Comment:
Remember, the protocol of mock api server is `https` but not `http`.

Reply:
If you use `http` protocol, you will get a response with `302 Redirect` without CORS access headers.